mtp_api/apps/account/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+from transaction.models import Transaction
+
+
+class FileType(models.Model):
```

Comment:
regarding file types: do you think it's resilient enough to assume that FileType pks are invariant because they're loaded in a fixture? they're used as the identifier in API requests, but do you think that lookups from client apps would need to be done each time a File object needs to be saved?

if the file type set is very rarely going to change, what're your thoughts on having `File.file_type` being a CharField with pre-defined choices?